dashboard/Dockerfile

```diff
@@ -1,18 +1,40 @@
-# RUN docker build . -t dashboard
```

Comment:
I'd throw this into ChatGPT just to see if it has any more optimizations:

I'd use this prompt:

"if you were an expert on best practices for a dockerfile, how would you make this one more efficient and abide by better conventions"

Reply:
same with the above dockerfile

Reply:
that file you sent had a couple of good optimizations. will add soon + test.

Reply:
tried a few of them but running into a couple of errors at build time (with npm, etc.)
I'll stick to the current implementation for now. using next.js [docs](https://github.com/vercel/next.js/blob/canary/examples/with-docker/Dockerfile) as inspiration.